BEIMA.Client/src/pages/Devices/DevicePage.js

```diff
@@ -224,22 +199,25 @@ const DevicePage = () => {
     const [docCopy, setDocCopy] = useState(documents)
     const [addedDocs, setAddedDocs] = useState([])
     const [removedDocs, setRemovedDocs] = useState([])
+    const navigate = useNavigate();
 
-    const submit = () => {
+    const updateDeviceCall = () => {
       const newDevice = {
-        tag:tag,
+        _id: deviceID,
+        deviceTypeId: deviceTypeID,
+        deviceTag:tag,
         manufacturer:manufacturer,
         modelNum:modelNum,
         serialNum:serialNum,
         yearManufactured:yearManufactured,
         notes:notes,
+        fields : fields,
```

Comment:
Should probably use the same spacing for all of these.

Reply:
Updated!